packages/terra-clinical-onset-picker/src/OnsetUtils.js

```diff
@@ -89,10 +89,6 @@ class OnsetUtils {
     const ageMoment = moment(birthdate).startOf('day'); // startOf to clear time from values
     const currentMoment = moment().startOf('day');
 
-    if (currentMoment.diff(ageMoment, 'weeks') === 0) {
-      return null;
```

Comment:
We should probably preserve returning null or return an empty array here. Otherwise a call to `allowedAgeUnits` can / will return age units that are not actually allowed.

Reply:
Theoretically this should never get called with an age less than a week old since we're removing the option in the OnsetPicker render. So whenever the age option is available, the week option would minimally always be there. That said, this didn't need to be removed either and does reinforce the correct "Allowed Age Units".

Reply:
updated here https://github.com/cerner/terra-clinical/pull/543/commits/cbaf09e53203239472adb85c23b4a658a641ae83